src/components/nav-sidebar/NavSidebar.stories.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
```

Comment:
what's this disabling?

Reply:
the linter was going off on the Link component not have a href but for this example it doesnt need one